### DISCUSSION TITLE
I don't like Vuex that much ....

### DISCUSSION THREAD

**Discussion by sadortun** [GoPlan-Finance/goplan-app#26] [General] [answered]
It may be a good idea to consider a more typescript friendly store I stead of using Vuex.

https://github.com/davestewart/vue-class-store

@GoPlan-Finance/core-team

**Comment by mkitzmann** [comment]
Very interesting! I quite like the Vuex dev-tools, but I have also had some headaches using vuex. This approach seems nice and simple, we can give it a try.

**Comment by sadortun** [comment, marked as answer]
~~Pinio~~ **pinia** is a mch better/mature option

**Reply by mkitzmann** [reply]
good to hear. Vuex 5 is going to be pretty much exactly like pinia, except for the cute pineapple :rofl:
